test/sagemaker_tests/mxnet/inference/conftest.py

```diff
@@ -168,6 +168,24 @@ def skip_eia_containers(request, docker_base_name):
         if 'eia' in docker_base_name:
             pytest.skip('Skipping eia container with tag {}'.format(docker_base_name))
 
+@pytest.fixture(autouse=True)
+def skip_neuron_containers(request, tag):
+    if request.node.get_closest_marker('skip_neuron_containers'):
```

Comment:
@aws-vrnatham You will need to add this to other test functions that don't use neuron images, such as those dedicated to EIA/CPU/GPU.